OpenEmu/pt.lproj/InfoPlist.strings

```diff
@@ -24,12 +24,12 @@
   SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
 
-"Archived Game" = "Archived Game"; // TODO
+"Archived Game" = "Jogo arquivado";
 
-"GameCorePlugin" = "GameCorePlugin"; // TODO
+"GameCorePlugin" = "PluginDoNucleo"; // TODO NOT SURE HOW THIS SHOULD BE TRANSLATED, CONTEXT?
```

Comment:
Hmm I'm not sure. I agree about 'plugin' but all references to "core" in the brazilian app are translated to "núcleo", besides that I think it will be weird two foreign words in the same sentence ('plugin' and 'core')